src/mds/CDir.h

```diff
@@ -646,7 +646,8 @@ class CDir : public MDSCacheObject, public Counter<CDir> {
       ceph::buffer::list &bl,
       int pos,
       const std::set<snapid_t> *snaps,
-      bool *force_dirty);
+      bool *force_dirty,
+      double rand_threshold);
```

Comment:
swap these two. output parameter should be at the end. Otherwise, looks good.

Reply:
done